tests/unit/ui/components/TestRating.js

```diff
@@ -142,13 +142,50 @@ describe(__filename, () => {
     verifyRating(root);
   });
 
+  it('renders an appropriate title when updating a rating', () => {
+    const root = render({ rating: 3 });
+
+    [1, 2, 3].forEach((rating) => {
+      expect(root.ratingElements[rating].className).toEqual('Rating-choice Rating-selected-star');
```

Comment:
Because this test is just for the `title`, we don't really need to assert anything about the `className` of the rating element.